### PR TITLE
add scRNA workshop in march

### PR DESCRIPTION
**Pull request by mtekman**
@erxleben Please review!

Also please check if the form link is accessible, and that the wording and dates make sense.

Thanks

### CODE REVIEW

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Preliminary schedule. Topics may be adjusted according to the participants demand
+
+| Day | Topics                                                                         |
+|:---:|:-------------------------------------------------------------------------------|
+| Mon | Pre-processing of scRNA-seq data and Barcoding                                 |
+| Tue | Downstream analysis with RaceID                                                |
+| Wed | Pre-processing of 10X scRNA-seq and QC with Scater                             |
+| Thu | Downstream analysis of 10X datasets with Scanpy, interactive tools and Jupyter |
+| Fri | Exercises, questions, discussions, bring your own data ... (until 14:00)       |
```
Comment (bgruening) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: I would remove the bring-your-own-data ...

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+## Program:
+
+Every day the workshop will run from 9:00-17:00 (give or take, depending on questions at the end). If the times will 
+change, it will be announced during the workshop.
+
+Preliminary schedule. Topics may be adjusted according to the participants demand
+
+| Day | Topics                                                                         |
+|:---:|:-------------------------------------------------------------------------------|
+| Mon | Pre-processing of scRNA-seq data and Barcoding                                 |
```
Comment (bgruening) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: Do you assume people can use Galaxy already? No 101 at the beginning?
Do you really need  6h for every of these tutorials?

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+* [Galaxy tipps and tricks](https://github.com/bgruening/galaxy-tricks)
+* [Galaxy training network](http://training.galaxyproject.org)
+
+## Lunch Options
+
+<iframe src="https://www.google.com/maps/d/embed?mid=13xIYbHTYlxxu-oopBTsSJHqK42M8lO6C" width="100%" height="480"></iframe>
+
+## Preparation
+
+You must go through two Galaxy interactive tours before beginning the training.
```
Comment (bgruening) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: Please go through ...

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+{% include map.html location=page.location showmap=true zoomlevel=15 hidepopup=true %}
+
+
+# Important notes
+
+1. If you are registered but **cannot attend** our workshop, please [contact us
+   by email](mailto:contact@usegalaxy.eu) immediately. We have a long waiting
+   list and can give your place to others, even on short notice.
+2. Please register to our [European Galaxy server](https://usegalaxy.eu) to perform the analysis. Participants from the
```
Comment (bgruening) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: I guess this can be removed.

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    lon: 7.845275
+supporters:
+- unifreiburg
+- denbi
+- mpi
+# special hiding of footer since we want to do it manually.
+hidefooter: true
+---
+
+We are offering a Galaxy workshop on single cell data analysis. 39 participants will be selected from all applicants.
```
Comment (bgruening) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: single-cell maybe bold?

File: _events/2020-01-20-GalaxyWS_scrna_FR.md (hunk 2 above)
Comment (mtekman) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: Is the 101 performed for every workshop? Do the RNA-seq trainings have the 101 on the first day? If so, I am happy to add

Comment (mtekman) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: I guess what I'm asking is: If they do the Galaxy 101, will they then need to do the RNA-seq tutorial, and only *then* they can do the single cell stuff?

Comment (bgruening) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: Yes, I think so. But the question is not what other workshops did, rather what do you expect from your participants ... should they know Galaxy already, or can people join that have never seen Galaxy.

Comment (mtekman) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: Okay, how about monday is Galaxy 101 and Mapping, tuesday is general RNA-seq, and then wed/thu/fri is scRNA-seq ?

https://github.com/usegalaxy-eu/website/pull/380/commits/08b03c4057f1f8f8b1a2c6e0e2aa6e4d07c80e26

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+---
+site: freiburg
+tags:
+- training
+title: Galaxy single cell RNA-seq data analysis workshop in Freiburg
+starts: 2020-03-23
+ends: 2020-03-27
+organiser:
+  name: Freiburg Galaxy Team & MPI Freiburg
```
Comment (erxleben) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: Is MPI FR involved?

File: _events/2020-01-20-GalaxyWS_scrna_FR.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  city: Freiburg im Breisgau
+  region: Baden-Württemberg
+  country: Germany
+  geo:
+    lat: 47.993469
+    lon: 7.845275
+supporters:
+- unifreiburg
+- denbi
+- mpi
```
Comment (erxleben) [inline _events/2020-01-20-GalaxyWS_scrna_FR.md]: if MPI is not involved, remove

File: _events/2020-01-20-GalaxyWS_scrna_FR.md (hunk 6 above)
Comment (bgruening) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: MPI is not involved ...

File: _events/2020-01-20-GalaxyWS_scrna_FR.md (hunk 7 above)
Comment (mtekman) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: should I keep deNBI?

Comment (erxleben) [reply _events/2020-01-20-GalaxyWS_scrna_FR.md]: SURE! Most important. You could add elixir as well because denbi is the German elixir node

**Comment by mtekman** [comment]
woo, thanks all!

**Comment by bgruening** [comment]
@mtekman can you please add the link to the workshop (the post) to the singlecell page and the google form. Thanks.

**Comment by mtekman** [comment]
will do

**Comment by mtekman** [comment]
Hmm, it seems I can't edit the google form without editing the url -- so I will have to update this PR again. I'm not sure if the url being edited means that a new form is generated...

**Comment by bgruening** [comment]
> Hmm, it seems I can't edit the google form without changing the url -- so I will have to update this PR again. I'm not sure if the url being changed means that a new form is generated...

this seems wrong ... this should be possible if you have the correct share address.

**Comment by mtekman** [comment]
Ah, maybe it's just the *shortened* url that changes then. Okay, will try.

**Pull request merged**